auto-spacing.txt

```diff
@@ -7,19 +7,27 @@
 #
 # @missing: 0000..10FFFF; O
 
-0030..0039     ; N  # Na  DIGIT ZERO..DIGIT NINE
-0041..005A     ; N  # Na  LATIN CAPITAL LETTER A..LATIN CAPITAL LETTER Z
+0021..0023     ; N  # Na  EXCLAMATION MARK..NUMBER SIGN
+0025..0027     ; N  # Na  PERCENT SIGN..APOSTROPHE
```

Comment:
Should [U+0027 APOSTROPHE](https://codepoints.net/U+0027) be the same category as QUOTATION MARK?

Reply:
Agreed.